test/unit/modules/blocks/utils.js

```diff
@@ -13,268 +13,558 @@
+						t_type: 3,
+					},
+					{
+						b_id: '6524861224470851795',
+						b_height: 1,
+						t_id: '3634383815892709956',
+						t_type: 2,
+					},
+					{
+						b_id: '6524861224470851795',
+						b_height: 1,
+						t_id: '3634383815892709957',
+						t_type: 0,
+					},
+				];
+
+				library.db.blocks.loadLastBlock = sinonSandbox
+					.stub()
+					.resolves(genesisBlock_votes);
+
+				blocksUtilsModule.loadLastBlock((err, block) => {
+					expect(block).to.be.an('object');
+					expect(block.id).to.equal('6524861224470851795');
+					expect(block.transactions).to.be.an('array');
+					expect(block.transactions[0]).to.be.an('object');
+					expect(block.transactions[0].id).to.equal('3634383815892709956');
+					expect(block.transactions[0].type).to.equal(2);
+					expect(block.transactions[1].id).to.equal('3634383815892709957');
+					expect(block.transactions[1].type).to.equal(0);
+					expect(block.transactions[2].id).to.equal('1465651642158264047');
+					expect(block.transactions[2].type).to.equal(3);
+					done();
 				});
+			});
 
-				describe('when transactionType equals SIGNATURE', () => {
-					it('should move it to the end of the block.transactions array');
+			it('should move signatures to the end', done => {
+				var genesisBlock_votes = [
+					{
+						b_id: '6524861224470851000',
+						b_height: 1,
+						t_id: '1465651642158264047',
+						t_type: 3,
+					},
+					{
+						b_id: '6524861224470851000',
+						b_height: 1,
+						t_id: '3634383815892709955',
+						t_type: 2,
+					},
+					{
+						b_id: '6524861224470851000',
+						b_height: 1,
+						t_id: '3634383815892709956',
+						t_type: 1,
+					},
+					{
+						b_id: '6524861224470851000',
+						b_height: 1,
+						t_id: '3634383815892709957',
+						t_type: 0,
+					},
+				];
+
+				library.db.blocks.loadLastBlock = sinonSandbox
+					.stub()
+					.resolves(genesisBlock_votes);
+
+				blocksUtilsModule.loadLastBlock((err, block) => {
+					expect(block).to.be.an('object');
+					expect(block.id).to.equal('6524861224470851000');
+					expect(block.transactions).to.be.an('array');
+					expect(block.transactions[0]).to.be.an('object');
+					expect(block.transactions[0].id).to.equal('1465651642158264047');
+					expect(block.transactions[0].type).to.equal(3);
+					expect(block.transactions[1]).to.be.an('object');
+					expect(block.transactions[1].id).to.equal('3634383815892709955');
+					expect(block.transactions[1].type).to.equal(2);
+					expect(block.transactions[2]).to.be.an('object');
+					expect(block.transactions[2].id).to.equal('3634383815892709957');
+					expect(block.transactions[2].type).to.equal(0);
+					expect(block.transactions[3]).to.be.an('object');
+					expect(block.transactions[3].id).to.equal('3634383815892709956');
+					expect(block.transactions[3].type).to.equal(1);
+					done();
 				});
 			});
 
-			it('should call modules.blocks.lastBlock.set with block');
+			it('should set the last block', done => {
```

Comment:
description is wrong - we are not testing if it will set last block, but only if it called modules.blocks.lastBlock.set with proper params (that function can do nothing for example)

Reply:
I think description matches the intended behavior, so not entirely wrong. But ok, we will use the original skeleton description, not sure why it was replaced in the first place.

Reply:
Done.